docs/en_US/TrialExample/RocksdbExamples.md

```diff
@@ -0,0 +1,68 @@
+# Tuning RocksDB on NNI
+
+## Overview
+
+[RocksDB](https://github.com/facebook/rocksdb) is a popular high performance embedded key-value database used in production systems at various web-scale enterprises including Facebook, Yahoo!, and LinkedIn.. It is a fork of [LevelDB](https://github.com/google/leveldb) by Facebook optimized to exploit many central processing unit (CPU) cores, and make efficient use of fast storage, such as solid-state drives (SSD), for input/output (I/O) bound workloads.
+
+The performance of RocksDB is highly contingent on its tuning. However, because of the complexity of its underlying technology and a large number of configurable parameters, a good configuration is sometimes hard to obtain. NNI can help to address this issue. NNI supports many kinds of tuning algorithms to search the best configuration of RocksDB, and support many kinds of environments like local machine, remote servers and cloud. By following this example, you are able to search the best configuration of RocksDB for a `fillrandom` benchmark with SMAC and TPE tuners. Other tuners can be easily adopted in the same way. Please refer to [here](../Tuner/BuiltinTuner.md) for more information.
+
+`code directory: example/trials/systems/rocksdb-fillrandom`
+
+## Goals
+
+This example illustrates how to use NNI to search the best configuration of RocksDB for a `fillrandom` benchmark supported by a benchmark tool `db_bench`, which is a official benchmark tool provided by RocksDB itself. Therefore, before running this example, please make sure NNI is installed and [`db_bench`](https://github.com/facebook/rocksdb/wiki/Benchmarking-tools) is in your `PATH`. Please refer to [here](../Tutorial/QuickStart.md) for detailed information about installation and preparing of NNI environment, and [here](https://github.com/facebook/rocksdb/blob/master/INSTALL.md) for compiling RocksDB as well as `db_bench`.
```

Comment:
About the environment, shall we explain more detail or offer a script to run it?(suggestion)
Which means users could run one command to start this example.

Reply:
The last section illustrated how to start this example. Shall we move it here?

Reply:
I mean the part that how to build the environments of RocksDB.

Reply:
I see~ Will add soon~

Reply:
Updated